src/index.html

```diff
@@ -4,10 +4,106 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" href="./style.css" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css" integrity="sha512-5A8nwdMOWrSz20fDsjczgUidUBR8liPYU+WymTZP1lmY9G6Oc7HlZv156XqnsgNUzTyMefFTcsFH/tnJE/+xBg==" crossorigin="anonymous" />
   <title>Table UI</title>
 </head>
 <body>
-  <h1>Table UI</h1>
+  <header>
+    <div class="left"><p>Show
+        <select id="hall">
+        <option>1</option>
+        <option>2</option>
+        <option>3</option>
+        <option>4</option>
+        <option>5</option>
+        <option>6</option>
+        <option>7</option>
+        <option selected>8</option>
+      </select>entries</p>
+    </div>
+      <div class="right">
+        <i class="fa fa-2x fa-search" aria-hidden="true"></i>
+        <input type="text" placeholder="Search" class="search">
```

Comment:
@jerineaso Input field should a `name` attribute as well.

Reply:
@satyambnsal Thank you so much sir for your kind feedback.